### PR TITLE
pleeeease i swear java isnt that bad pls rebase me pls pls pls pls

### PR DESCRIPTION
**Pull request by EvanCPSC**
## supporting arguments:

image

### CODE REVIEW

File: helo.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Main {
```
Comment (amyipdev) [inline helo.java]: `public class`

**Review by amyipdev — CHANGES_REQUESTED** [review]

File: helo.java (hunk 1 above)
Comment (EvanCPSC) [reply helo.java]: i might

Comment (EvanCPSC) [reply helo.java]: ok i did it https://cdn.discordapp.com/emojis/1100919837414395904.gif?size=128&quality=lossless

**Comment by EvanCPSC** [comment]
im sorry for using java

**Pull request closed without merging**